frontend/web/components/RolesTable.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { FC } from 'react'
```

Comment:
Moved to reduce bloat on org settings, this will also be needed to support https://github.com/Flagsmith/flagsmith/issues/3619